src/Resources/views/WishlistDetails/_variantPrice.html.twig

```diff
@@ -5,20 +5,17 @@
 {% set prices %}
     [
     {%- for price in pricing -%}
-        {
+        {{- '{' -}}
         {%- for option, value in price -%}
             "{{- option -}}":
             {%- if option == 'value' or option == 'original-price' -%}
                 "{{ money.convertAndFormat(value) -}}"
-            {%- elseif not value is iterable -%}
-                "{{ value|replace({'\"': '\''}) -}}"
             {%- else -%}
-                ""
+                "{{ value }}"
```

Comment:
We cannot just display the value, as there are i.e. arrays of CatalogPromotions etc.